CanvasCore/CanvasCore/Helm/HelmViewController.swift

```diff
@@ -240,41 +199,40 @@ public final class HelmViewController: UIViewController, HelmScreen, PageViewEve
     
     // MARK: - Styles
     @objc public func handleStyles() {
-        if let title = screenConfig[PropKeys.title] as? String {
-            if let subtitle = screenConfig[PropKeys.subtitle] as? String, subtitle.count > 0 {
-                if (twoLineTitleView == nil) {
-                    twoLineTitleView = self.titleView(with: title, and: subtitle, given: screenConfig)
-                    twoLineTitleView?.isAccessibilityElement = true
-                    twoLineTitleView?.accessibilityTraits = UIAccessibilityTraits.header
-                    navigationItem.titleView = twoLineTitleView
-                    navigationItem.title = nil
-                }
-                
-                if let titleView = twoLineTitleView, let titleLabel = titleView.titleLabel, let subtitleLabel = titleView.subtitleLabel {
-                    styleTitleViewLabels(titleLabel: titleLabel, subtitleLabel: subtitleLabel)
-                    titleLabel.text = title
-                    subtitleLabel.text = subtitle
-                    titleView.accessibilityLabel = "\(title), \(subtitle)"
-                }
+        switch screenConfig[PropKeys.navBarStyle] as? String {
+        case "context":
+            if let color = screenConfig.navBarColor {
+                navigationController?.navigationBar.useContextColor(color)
             }
-            self.title = title
+        case "global":
+            navigationController?.navigationBar.useGlobalNavStyle()
+        default:
+            navigationController?.navigationBar.useModalStyle()
```

Comment:
This is the real change in strategy for HelmViewController. Instead of each component sending a full list of navigation bar properties, they can send a `navBarStyle` of `"global"`, `"context"`, or `"modal"` and if using `"context"`, then the `navBarColor` is used too.